.github/workflows/integration_test.yaml

```diff
@@ -1,15 +1,15 @@
 name: Integration tests
-
 on:
   pull_request:
   workflow_call:
-
 jobs:
   integration-tests:
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    uses: arjun11-malik/operator-workflows/.github/workflows/integration_test.yaml@main
     secrets: inherit
     with:
-      channel: 1.28-strict/stable
+      provider: k8s
+      use-canonical-k8s: true
+      channel: edge
```

Comment:
suggestion, let's be explicit with the track (e.g. `latest/edge` or `1.28-strict/edge`

also, any reason why we're testing against k8s edge risk? and not the stable risk?